### PR TITLE
Otp input view

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: src/main/resources/templates/otp.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+          integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <title>OTP page</title>
+</head>
+<body>
+<div class="container pt-5 pb-5 d-md-flex justify-content-center">
+    <form method="post" action="/page/otp" class="form-inline">
+        <div class="form-group raw flex-nowrap" style="width: 65vh; justify-content: space-around">
+            <label for="otp" class="mr-3 h5">Please enter the OTP received from bank:</label>
```
Comment (spetrychenko) [inline src/main/resources/templates/otp.html]: I think you have missed the article `the` before bank word.

File: src/main/resources/templates/otp.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <input name="otp" id="otp" type="password" class="form-control mr-3 col-sm-5">
```
Comment (spetrychenko) [inline src/main/resources/templates/otp.html]: not sure about type of the field, let's see what the team voted it the slack channel

File: src/main/resources/templates/otp.html (hunk 1 above)
Comment (berezkin88) [reply src/main/resources/templates/otp.html]: yeah, good spot, thx

File: src/main/resources/templates/otp.html (hunk 2 above)
Comment (berezkin88) [reply src/main/resources/templates/otp.html]: to me, it sounds like sensitive data as banks usually ask to keep such codes from exposing to anyone

**Pull request merged**